game/scripts/vscripts/modifiers/modifier_aura_item_upgrade.lua

```diff
@@ -8,8 +8,12 @@ function modifier_aura_item_upgrade:IsPurgable()
   return false
 end
 
-function modifier_aura_item_upgrade:GetAttributes()
-  return MODIFIER_ATTRIBUTE_MULTIPLE
```

Comment:
Why was this removed? I'm pretty sure it is 100% necessary. Without it, it won't allow multiple instances of the modifier on the hero and it will probably not work properly when items are combined in quick succession.

Reply:
You are right

Reply:
Truth, I only tested with 1 item at a time.. Easiest way to test is get everything ready for combining in stash then walk into the other well. Items will instant combine and you die a quarter second later.